### ISSUE TITLE
alt text for that Image should be more descriptive

### ISSUE THREAD

**Issue by AhmedAdelSelim** [Saytara2001/Udemy-Website#2]
1- alt text for that Image should be more descriptive 
     for each subject i know it is a photo for the "subject" but the person who has problem with internet connection and won't able to see the      photo need what is this photo was .
